src/app/components/CollapsibleNavigation/index.tsx

```diff
@@ -131,7 +131,10 @@ const CollapsibleNavigation = ({
                           <a
                             href={link.href}
                             css={styles.subNavLink}
-                            aria-labelledby={linkLabelId}
+                            {...(link?.lang && { lang: link.lang })}
+                            {...(link?.latinTransliteration && {
+                              'aria-label': link?.latinTransliteration,
```

Comment:
Could this have some unit tests please to ensure lang and latinTranslieration are included as required?